library/src/main/java/com/github/amlcurran/showcaseview/ShowcaseView.java

```diff
@@ -343,6 +340,24 @@ public boolean onTouch(View view, MotionEvent motionEvent) {
     }
 
     private static void insertShowcaseView(ShowcaseView showcaseView, Activity activity) {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+            Display display = ((WindowManager) activity.getSystemService(Context.WINDOW_SERVICE)).getDefaultDisplay();
+            int myOrientation = display.getRotation();
+            switch (myOrientation) {
+                case Surface.ROTATION_0:
+                    showcaseView.setPadding(0, 0, 0, ApiUtils.getSoftButtonsBarSizePort(activity));
+                    break;
+                case Surface.ROTATION_90:
+                    showcaseView.setPadding(0, 0, ApiUtils.getSoftButtonsBarSizeLand(activity), 0);
+                    break;
+                case Surface.ROTATION_180:
+                    showcaseView.setPadding(0, 0, ApiUtils.getSoftButtonsBarSizeLand(activity), 0);
```

Comment:
So I was looking into [onApplyWindowInsetListener](https://developer.android.com/reference/android/view/View.html#setOnApplyWindowInsetsListener(android.view.View.OnApplyWindowInsetsListener)) to the view, but it never gets called.  I'm not entirely sure how to use it yet, but that replaces `fitSystemWindows`.  That might be a good option as you don't have to worry about checking orientation and determining where the nav bar is (bottom on tablet landscape, right on phone landscape).